.env

```diff
@@ -34,4 +34,6 @@ EXTERNAL_INDEX_INTERVAL=43200
 INDEX_CACHE_PATH=/tmp/modrinth-id-cache.json
 
 GITHUB_CLIENT_ID=3acffb2e808d16d4b226
-GITHUB_CLIENT_SECRET=none
\ No newline at end of file
+GITHUB_CLIENT_SECRET=none
+
+IP_SALT=labrinth
```

Comment:
This isn't what I was talking about; this needs to be generated randomly on server startup, or it isn't useful at all.  If it's static, even if it's configured to be something else on the production host, then it is just as easy to brute force as without it.

This isn't as much of an issue with older downloads being cleared, but this doesn't do much as is - it should be easy to add a random number to the end on startup and then pass that to the routes using actix's shared data.